app/src/main/java/com/pajato/android/gamechat/chat/ContactManager.java

```diff
@@ -64,9 +70,9 @@
 
     // Public instance methods.
 
-    /** Return a list of device contacts formatted as chat list items. */
-    public List<ListItem> getDeviceContactList() {
-        // Convert each contact to a chat list item.
+    /** Return a list of device contacts formatted as list items. For future use. */
+    @SuppressWarnings("unused") public List<ListItem> getDeviceContactList() {
```

Comment:
This is actually a much better way to silence AS than to pull code now only to maybe resurrect it later.